samples/runtime/autosubstitution/README.md

```diff
@@ -0,0 +1,41 @@
+---
+page_type: sample
+languages:
+- qsharp
+products:
+- qdk
+description: "This sample uses the `Microsoft.Quantum.AutoSubstitution` NuGet package to provide alternative operations based on the simulator that is used."
+---
+
+# Using the AutoSubstitution rewrite step
+
+The `Microsoft.Quantum.AutoSubstitution` NuGet package offers a rewrite step
+that can link an operation `Op` to a set of pairs of an alternative operations
+`AltOp` and a simulator `Sim`, such that `AltOp` is executed as a replacement
+for `Op` when being invoked in `Sim`.
```

Comment:
```suggestion
The `Microsoft.Quantum.AutoSubstitution` NuGet package offers a decorator `@SubstitutableOnTarget` that lets you replace one operation with another when using a particular simulator. For instance, decorating the operation `Op` with `@SubstitutableOnTarget("AltOp", "Sim")` lets the compiler perform a rewrite step that can link an operation `Op` to a set of pairs of an alternative operations
`AltOp` and a simulator `Sim`, such that `AltOp` is executed as a replacement
for `Op` when being invoked in `Sim`.
```

Reply:
Thanks! I just replaced _decorator_ with _attribute_ to use Q# naming.

Reply:
Technically, `@SubstitutableOnTarget` isn't a decorator, but an attribute, since it only adds metadata to the operation that is later consumed by the auto-substitution rewrite step.